src/edu/psu/sweng500/team8/coreDataStructures/Cell.java

```diff
@@ -104,4 +104,41 @@ public void load(BinaryInputStream stream) throws IOException {
 			this.type = ValueType.Given;
 		}
 	}
+
+	@Override
+	public boolean equals(Object o) {
+
+		if (null == o) {
+			return false;
+		}
+
+		if (o instanceof Cell) {
+
+			if (this.coordinates.getRowIndex() != ((Cell) o).coordinates

```

Comment:
Yes. Again, it all depends which school of best practices you follow. JTest static code analysis used to warn against having a one long conditional statement for a sake of reducing the number of lines, so I am just following that. We can definitely reduce the number of lines by merging two or maybe three conditions together into a single line which isn't giving us that much of benefit anyway.